nesta/packages/mag/parse_abstract.py

```diff
@@ -13,8 +13,7 @@ def _uninvert_abstract(inverted_abstract):
     for term, idxs in inverted_abstract['InvertedIndex'].items():
         for idx in idxs:
             terms[idx] = term
-    if None in terms:
```

Comment:
this was originally here to catch if such edge cases existed. I found them :p and it turns out that although MAG inverted abstracts have "missing" indexes, from inspection of the "real" articles on e.g. biorxiv that they don't represent anything, i.e. they can be omitted, which is what you will see in the new code